GiniVision/Classes/Core/GiniConfiguration.swift

```diff
@@ -8,6 +8,16 @@
 
 import UIKit
 
+@objc public class GiniColor : NSObject {
+    var lightModeColor: UIColor = .white
```

Comment:
Since we don't have an initializer with no arguments we don't have initialize the variables. They will be always get a value in the initializer.